### ISSUE TITLE
SVG icons should have accessible text

### ISSUE THREAD

**Issue by david-crespo** [oxidecomputer/console#2322]
They could probably come from `@oxide/design-system` with built-in default names matching their component names.

<img width="1263" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/ac741442-2b2f-4f5c-9fb4-4ac57943fcba">

**Comment by charliepark** [comment]
Just to log some notes on this, I explored this for a bit this week and was working on a refactor of icons in the design-system repo ([here](https://github.com/oxidecomputer/design-system/tree/icon-refactor)). Crespo noted that these files are actually generated by [this executable markdown file](https://github.com/oxidecomputer/design-system/blob/icon-refactor/scripts/export-icons.md). Next would be to explore in Figma, to see what attributes we can add to the SVGs themselves, or, if that's too painful, to see about updating that executable .md file.

Two quick notes: as the icons are used "decoratively", we should be able to set them to have `role="presentation"`, and can hide them with `aria-hidden`. Everywhere we have them, we either have text announcing, say, the page title, or are using them in a button (like CopyToClipboard) that has its own aria label.
